CMakeLists.txt

```diff
@@ -444,3 +444,13 @@ if (MPI)
       LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
       RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
 endif ()
+
+# Provide FindArpackNG to users.
+configure_file(ArpackNGConfig.cmake.in "${PROJECT_BINARY_DIR}/ArpackNGConfig.cmake")
+install(FILES "${PROJECT_BINARY_DIR}/ArpackNGConfig.cmake" DESTINATION "${CMAKE_INSTALL_PREFIX}")
+set(ArpackNG_MAJOR_VERSION 3)
```

Comment:
Is that mandatory to split MAJOR/MINOR/PATCH ?
why not having 3.5.0 in arpackng_version?

Reply:
No. I did like people usually do.